src/main/java/com/google/sps/managers/MockDatastoreManager.java

```diff
@@ -178,4 +178,8 @@ private Game getRandomGame() {
         res.setTotalDifficulty(getRandomIntegerBetween(numDifficultyVotes, 3*numDifficultyVotes));
         return res.build();
     }
+
+    public boolean doesUsernameExist(String username) {
```

Comment:
Put a TODO so you dont forget or if I'm looking at ur code I know that this is still not implemented.

Reply:
This is done. "return false" is all I need since this is just a mock class.